commons-dlp/src/main/resources/db/changelog/exo-dlp.db.changelog-1.0.0.xml

```diff
@@ -98,4 +98,16 @@
     <createSequence sequenceName="SEQ_RESTORED_DLP_ITEMS_ID" startValue="1" />
   </changeSet>
 
+  <changeSet author="exo-dlp" id="1.0.0-10">
+    <modifyDataType columnName="KEYWORDS"
+                    newDataType="LONGTEXT"
+                    tableName="DLP_POSITIVE_ITEMS"/>
+    <rollback>
+      <!-- This rollback is need because in unit test we will revert all changes after each test case-->
```

Comment:
if it's used in Unit Test only, you can add a dedicated changeset in test resources only. But, I think that we don't need this change to be applied even in unit tests.